docs/DeveloperGuide.md

```diff
@@ -4,30 +4,148 @@
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
+* [JSON-java](https://github.com/stleary/JSON-java)
+* [three-ten-extra](https://www.threeten.org/threeten-extra/)
+* * Requesting and Parsing of data from API into Java
+* https://www.youtube.com/watch?v=lDEfoSwyYFg
```

Comment:
I believe it would be beneficial if this link was a hyperlink, making it easier for readers to click on it.